internal/file/tar.go

```diff
@@ -93,3 +95,13 @@ func UnTarGz(dst string, r io.Reader) error {
 		}
 	}
 }
+
+func copyWithLimits(writer io.Writer, reader io.Reader, byteReadLimit int64, target string) error {
```

Comment:
I've had a bit of trouble figuring out what `target` is, after looking at this function and its tests. Is this a file path?

Reply:
it is a path within the archive, I'll adjust the name